### PR TITLE
Translate & update channel-stats.md

### PR DESCRIPTION
**Pull request by Fridolin012**
## Type of PR

- [X] Translation update / creation
- [ ] Original file creation
- [ ] Original file content (not typo) update
- [X] Original file typo update

## Checks

If you have changed the content (/ meaning) of an original file, please confirm:

- [ ] Yes, I have copied the updated section to all translations of this file manually.

## Notes

Please enter additional information here.

### CODE REVIEW

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Channel-Statistiken
+
+Erstelle Kanäle mit Statistiken über deinen Server - automatisch aktualisiert.
+
+<ModuleOverview moduleName="channel-stats" />
+
+## Funktionen {#features}
+
+Erstelle Sprachkanäle auf deinem Server, welche automatisch aktuelle Statistiken anzeigen - wie die Anzahl der Mitglieder
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md:9]:
```suggestion
Erstelle Sprachkanäle auf deinem Server, welche automatisch aktuelle Statistiken anzeigen, wie z.B. die Anzahl der Mitglieder 
```
In german, the use of "wie", associated with "-" as an expression of a exemplary value, sounds weird

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+Erstelle Kanäle mit Statistiken über deinen Server - automatisch aktualisiert.
+
+<ModuleOverview moduleName="channel-stats" />
+
+## Funktionen {#features}
+
+Erstelle Sprachkanäle auf deinem Server, welche automatisch aktuelle Statistiken anzeigen - wie die Anzahl der Mitglieder 
+auf deinem Server, die eine bestimmte Rolle haben.
+
+Hier ist ein Beispiel, wie 3 Sprachkanäle einige Statistiken über einen Server zeigen:
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md:12]:
```suggestion
Hier ist ein Beispiel von drei Sprachkanäle, die einige Statistiken über einen Server zeigen:
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+![](@site/docs/assets/custom-bot/modules/channel-stats/example.png)
+
+## Einrichtung {#setup}
+
+1. Erstelle einen Kanal, den du für die Statistiken verwendet willst (wir empfehlen das Benutzen einer Kategorie oder eines Sprachkanals).
+2. Stelle sicher, dass dein Bot die Berechtigungen "Kanal sehen" und "Kanal verwalten" auf den Kanal hat.
+3. Öffne die [Channel-Statistiken-Konfiguration](https://scnx.app/de/glink?page=bot/configuration?file=channel-stats%7Cchannels).
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md:20]:
```suggestion
3. Öffne die [Kanal-Statistiken-Konfiguration](https://scnx.app/de/glink?page=bot/configuration?file=channel-stats%7Cchannels).
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# Channel-Statistiken
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md:1]:
```suggestion
# Kanal-Statistiken
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+|---------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Kanal                     | Das ist der Kanal, welcher als Statistik-Kanal genutzt wird. Sein Name wird automatisch zum Wert von "Kanalname" aktualisiert.                                         |
+| Kanalname                 | Das ist der Name, zu dem der Kanal automatisch aktualisiert wird. <br/><i>Bitte überprüfe verfügbare Parameter in deinem Dashboard.</i>                                | 
+| Aktualisierungsintervall  | Das ist das Intervall in Minuten, in welchem dein Bot den Name des Kanals aktualisiert. Aufgrund technischen Limititationen Discords muss dies höher als sieben sein.  |
+
+## Fehlerbehebung {#troubleshooting}
+
+<details>
+<summary>Ich versuche den Namen des Kanals zu ändern, doch der Bot ändert ihn zurück</summary>
+    Wenn du den Namen des Kanals verändern willst, musst du das Feld "Kanalname" in der <a href="https://scnx.app/glink?page=bot/configuration?file=channel-stats%7Cchannel">Modul-Konfiguration</a> des Kanals aktualisieren.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md:45]:
```suggestion
    Wenn du den Namen des Kanals verändern willst, musst du das Feld "Kanalname" in der <a href="https://scnx.app/de/glink?page=bot/configuration?file=channel-stats%7Cchannel">Modul-Konfiguration</a> des Kanals aktualisieren.
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+</details>
+
+<details>
+    <summary>Mein Kanal wird nicht aktualisiert</summary>
+    <ul>
+        <li>Stelle sicher, dass du das festgelegte Aktualisierungs-Intervall abgewartet hast, da der Kanalname nur aktualisiert wird, wenn das Intervall abgelaufen ist.</li>
+        <li>Es dauert eventuell länger als das für diesen Kanal festgelegte Aktualisierungs-Intervall aufgrund Limitierungen von Discord. Bitte warte bis zu drei Stunden.</li>
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md:52]:
```suggestion
        <li>Es dauert eventuell, aufgrund von Limitierungen von Discord, länger als das für diesen Kanal festgelegte Interval. Bitte warte bis zu drei Stunden.</li>
```
This is way more readable I think

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+        <li>Stelle sicher, dass du den korrekten Parameter Wort für Wort (Groß- und Kleinschreibung beachten!) eingegeben hast. Ungültige Parameter werden nicht ersetzt.</li>
+        <li>Stelle sicher, dass dein Bot die Berechtigungen "Kanal sehen" und "Kanal verwalten" auf den festgelegten Kanal hat.</li>
+        <li>Aktualisiere deinen Client, wenn eine Änderung nicht sichtbar ist.</li>
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md:55]:
```suggestion
        <li>Aktualisiere deinen Discord-Client, wenn eine Änderung nicht sichtbar ist.</li>
```

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+    </ul>
+    Der Name des Channels wird nicht aktualisiert, wenn der neue dem altem gleicht.
```
Comment (SCDerox) [inline i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md:57]:
```suggestion
    Der Name des Channels wird nicht aktualisiert, wenn der neue dem Altem gleicht.
```

**Review by SCDerox — CHANGES_REQUESTED** [review]
Thanks so much for your contribution - I've found a few issues, please review my comments.

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md (hunk 6 above)
Comment (Fridolin012) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md:52]:
```suggestion
        <li>Es dauert eventuell, aufgrund von Limitierungen von Discord, länger als das für diesen Kanal festgelegte Intervall. Bitte warte bis zu drei Stunden.</li>
```

I think “interval” is spelled with 2 “l”.

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md (hunk 2 above)
Comment (Fridolin012) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md:12]:
```suggestion
Hier ist ein Beispiel von drei Sprachkanälen, die einige Statistiken über einen Server zeigen:
```

I think "voice channels" has to be plural

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md (hunk 6 above)
Comment (SCDerox) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md:52]: This is correct, I've updated my comment

File: i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md (hunk 2 above)
Comment (SCDerox) [reply i18n/de/docusaurus-plugin-content-docs/current/custom-bot/modules/administration/channel-stats.md:12]: this is correct, I've updated my comment

**Review by SCDerox — APPROVED** [review]
lgtm

**Comment by SCDerox** [comment]
Thanks for your contribution - you have received a payout as part of our [Open Source Developer Program](https://faq.scnx.app/open-source-developer-pool/). You can find details in your SCNX organisation payout history. To pay out the received amount, please follow instructions in your dashboard. If you have any questions, please contact ScootKit staff on [scnx.app/help](https://scnx.app/help).
Cheers ✌️

**Comment by Fridolin012** [comment]
Kommentar

**Comment by SCDerox** [comment]
Thanks for getting back to me. It seems like you've already received a bounty for this contribution.

**Comment by Fridolin012** [comment]
Oh, I overlooked that.

**Pull request merged**